services/src/pro/users/oidc.rs

```diff
@@ -122,6 +122,7 @@ pub enum OidcError {
     IllegalProviderConfig {
         source: ParseError,
     },
+    #[snafu(display("ProviderDiscoveryError: {}", source))]
```

Comment:
why is this enum still ending in "Error" when you removed it in the other cases?

Reply:
The enum is not, this is just the message.